searx/templates/oscar/base.html

```diff
@@ -89,6 +89,10 @@
                     <a href="{{ brand.GIT_URL }}">{{ _('Source code') }}</a> |
                     <a href="{{ brand.ISSUE_URL }}">{{ _('Issue tracker') }}</a> |
                     <a href="{{ brand.PUBLIC_INSTANCES }}">{{ _('Public instances') }}</a>
+                    {% if contact_url %}
+                    |
+                    <a href="{{ contact_url }}">{{ _('Contact Instance Maintainer') }}</a>
```

Comment:
Why the uppercase for the words "instance" and "maintainer"?

Reply:
See comment: https://github.com/searx/searx/pull/2391#issuecomment-745955309

`<a href="{{ brand.CONTACT_URL }}">{{ _('Contact Instance Maintainer') }}</a>`

Reply:
9a2be39